content/posts/master-class-dynamical-systems-kth.md

```diff
@@ -0,0 +1,27 @@
++++
+categories = ["news", "events"]
+date = "2024-01-07 14:30:00"
+title = "Master Class in dynamical systems at KTH Stockholm"
++++
+
+The [Department of Mathematics at KTH](https://www.kth.se/math) *(Royal Institute of 
+Technology)* announces their second Master Class in Dynamical Systems, which will be 
+held in Stockholm from May 20 to May 24, 2024. 
+
+The Master class consists of three courses, 5 lectures each:
+
+*"Partially hyperbolic diffeomorphisms and group actions"*, by Danijela 
+Damjanovic (KTH) and Amie Wilkinson (University of Chicago);  
+*"Geometric methods in Arnold diffusion: old and new results"*, by Tere Seara 
+(Universitat Politecnica de Catalunya);  
+*“Conjugacy classes of real analytic maps”*, by Sebastian Van Strien (Imperial 
+College). 
+
+The Master Class is aimed at Master's and PhD students, as well as postdocs. Limited 
+funding is available to cover accommodation for participants. Travel costs are not 
+supported.
+
+All interested students should **apply by February 15, 2024** at 
+<https://www.kth.se/form/master-class-dynamical-systems-2024>.
```

Comment:
I don't know if this syntax is supported by `hugo`, the website generator. We will find out soon :)

Reply:
Works fine!